draft-packages/button/docs/styles.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "~@kaizen/component-library/styles/responsive";
+
+.buttonSection {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  align-items: flex-start;
+  @include ca-media-mobile {
+    flex-direction: column;
+    margin-bottom: 1rem;
+  }
+}
+.buttonState {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-right: 5rem;
+  @include ca-media-mobile {
+    flex-direction: column;
+    margin-top: 1rem;
+  }
+}
```

Comment:
No change requested for this PR, but when we do this to another component, I'd love to think about how to make these re-usable so they're consistent across all our stories.

Perhaps they're new components for storybook only? Perhaps they're styles that we re-use in all of our stories?

Reply:
Thats a really good idea. I'll add that as another issue to create one and then refactor the buttons stories after.